presto-jdbc/src/main/java/com/facebook/presto/jdbc/PrestoConnection.java

```diff
@@ -118,7 +118,8 @@
         this.connectionProperties = uri.getProperties();
         this.queryExecutor = requireNonNull(queryExecutor, "queryExecutor is null");
 
-        timeZoneId.set(TimeZone.getDefault().getID());
+        timeZoneId.set(uri.getTimeZoneID().isPresent() ?
```

Comment:
This can be updated based on the changes to the method `getTimeZoneID`

Reply:
Done. Thanks.